engine/src/main/java/org/terasology/logic/console/commands/CoreCommands.java

```diff
@@ -401,6 +404,75 @@ public String spawnBlock(@Sender EntityRef sender, @CommandParam("blockName") St
         return "Spawned block.";
     }
 
+
+
+	//Works on windows only , i don't know if you can even play the game on other platforms
+    @Command(shortDescription = "Your ping to the server", helpText = "The time it takes the packet " +
+    "to reach the server and back" , runOnServer = false, requiredPermission = PermissionManager.NO_PERMISSION)
+    public String ping(@Sender EntityRef sender ) {
+        //The class is just to tidy up the code , and not having random methods in this file
+        class InnerClass{
```

Comment:
Since none of the methods are called more than once, this inner class only makes the function slightly longer. I'd move this code into the `ping` function or `private` helper methods within the `CoreCommands` class.